rxbindings/README.md

```diff
@@ -1,64 +1,49 @@
-# amplify-android-rxbindings
-A wrapper for the AWS Amplify Android framework, which exposes
-functionalities via RxJava2
+# Rx Bindings
 
-Instead of using Amplify's native callback interface, you can interact
-with Amplify APIs by means of RxJava2
+The Rx Bindings provide a facade to the Amplify library.  Instead of
+using Amplify's native callback interface, you can interact with Amplify
+APIs by means of RxJava2
 [`Observable`](http://reactivex.io/RxJava/javadoc/io/reactivex/Observable.html),
 [`Single`](http://reactivex.io/RxJava/javadoc/io/reactivex/Single.html), and
 [`Completable`](http://reactivex.io/RxJava/javadoc/io/reactivex/Completable.html).
 
+For more information see [the public documentation](https://docs.amplify.aws/lib/project-setup/rxjava/q/platform/android)
+and [this blog post](https://aws.amazon.com/blogs/mobile/using-rxjava-with-aws-amplify-android-library/).
+
 ## Usage
 
 ### Pre-requisites
 
-As a pre-requisite, deploy AWS resources to your backend using the
-Amplify CLI, as in the [Getting
-Started guide](https://aws-amplify.github.io/docs/android/start).
-
+All of the pre-requisites in the [Getting
+Started guide](https://aws-amplify.github.io/docs/android/start) apply
```

Comment:
When navigating to https://aws-amplify.github.io/docs/android/start, I'm redirected to https://docs.amplify.aws/start/q/integration/android. Should we use https://docs.amplify.aws/start/q/integration/android as the link instead?

Reply:
Sounds good!